gfx_window.cpp

```diff
@@ -115,12 +121,20 @@ class GfxWindowInternal
                              SWP_NOMOVE | SWP_NOSIZE | SWP_NOZORDER | SWP_NOOWNERZORDER | SWP_FRAMECHANGED);
             }
         }
+        else
+        {
+            RECT window_rect = { 0, 0, (LONG)window_width_, (LONG)window_height_ };
+            UINT dpi = GetDpiForWindow(window_);
+            AdjustWindowRectExForDpi(&window_rect, window_style, FALSE, window_style_ex, dpi);
+            SetWindowPos(window_, NULL, 0, 0, window_rect.right - window_rect.left,
+                window_rect.bottom - window_rect.top, SWP_NOZORDER | SWP_NOACTIVATE | SWP_NOMOVE);
+        }
```

Comment:
`AdjustWindowRect` is now DPI aware. This requires moving the window sizing till after window creation as you need a valid window to get the associated monitors DPI scaling.